src/test/resources/logback-test.xml

```diff
@@ -13,6 +13,8 @@
         </encoder>
     </appender>
 
+    <logger name="org.constellation" level="debug" />
```

Comment:
shouldn't you change the root level of ref="console"?

Reply:
I think we don't need other packages except this ?

Reply:
Yep, you're right